sim/play.py

```diff
@@ -78,40 +89,108 @@ def play(args: argparse.Namespace) -> None:
+                    shape=[(env_cfg.env.frame_stack - 1) * (11 + env.num_dof * 3)],
+                    description="Buffer of previous observations"
+                )
+            ),
+            P.ValueSchema(
+                value_name="vector_command",
+                vector_command=P.VectorCommandSchema(
+                    dimensions=3,  # x_vel, y_vel, rot
+                )
+            ),
+            P.ValueSchema(
+                value_name="timestamp",
+                timestamp=P.TimestampSchema(
+                    unit=P.TimestampUnit.SECONDS,
+                    description="Current policy time in seconds"
+                )
+            ),
+            P.ValueSchema(
+                value_name="joint_positions",
+                joint_positions=P.JointPositionsSchema(
+                    joint_names=JOINT_NAMES,
+                    unit=P.JointPositionUnit.RADIANS,
+                )
+            ),
+            P.ValueSchema(
+                value_name="joint_velocities", 
+                joint_velocities=P.JointVelocitiesSchema(
+                    joint_names=JOINT_NAMES,
+                    unit=P.JointVelocityUnit.RADIANS_PER_SECOND,
+                )
+            ),
+            P.ValueSchema(
+                value_name="previous_actions",
+                joint_positions=P.JointPositionsSchema(
+                    joint_names=JOINT_NAMES,
+                    unit=P.JointPositionUnit.RADIANS
+                )
+            ),
+            # Abusing the IMU schema to pass in euler and angular velocity instead of raw sensor data
+            P.ValueSchema(
+                value_name="imu_angular_velocity",
+                imu=P.ImuSchema(
+                    use_accelerometer=False,
+                    use_gyroscope=True,
+                    use_magnetometer=False,
+                )
+            ),
+            P.ValueSchema(
+                value_name="imu_orientation",
+                imu=P.ImuSchema(
+                    use_accelerometer=True,
+                    use_gyroscope=False,
+                    use_magnetometer=False,
+                )
+            )
+        ]
+    )
+
+    output_schema = P.IOSchema(
+        values=[
+            P.ValueSchema(
+                value_name="joint_positions",
+                joint_pos=P.JointPosSchema(
+                    joint_names=JOINT_NAMES,
+                    unit=P.JointPosUnit.RADIANS,
+                ),
+            )
+        ]
+    )
 
-        export_to_onnx(actor_model, input_tensors=input_tensors, config=export_config, save_path="kinfer_policy.onnx")
-        print("Exported policy as kinfer-compatible onnx to: ", path)
+    # Create the full model schema
+    model_schema = P.ModelSchema(
+        input_schema=input_schema,
+        output_schema=output_schema
+    )
 
+    if args.export_onnx:
+        kinfer_policy = export_model(
+            model=policy,
```

Comment:
the policy module will need to be update to handle the new input format

Reply:
you want to run the policy in isaacgym through kinfer?